darwin/dataset/remote_dataset.py

```diff
@@ -19,10 +19,10 @@ class RemoteDataset:
     def __init__(
         self,
         *,
+        team: str,
         name: str,
         slug: Optional[str] = None,
```

Comment:
we could make slug a @property which calls a _slugify (xD) method on name. In this way we can ensure consistency between name and slug by design

Reply:
probably is that the slug could been changed later on so the name and slug doesn't match any more.